contents/templates/mobile-dashboard/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Mobile dashboard
+subtitle: Requires iOS, Android, Flutter, or React Native SDK
+thumbnail: ../thumbnails/mobile.png
+featuredImage: ../thumbnails/featured/real-time.png
```

Comment:
Doesn't matter too much, but do you want a mobile feature image here?

Reply:
Eh, I figured this would do vs the time to make one tbh